vsintegration/src/FSharp.Editor/LanguageService/SymbolHelpers.fs

```diff
@@ -67,69 +66,79 @@ module internal SymbolHelpers =
             return symbolUses
         }
 
-    let getSymbolUsesInProjects
-        (
-            symbol: FSharpSymbol,
-            projects: Project list,
-            onFound: Document -> range -> Async<unit>,
-            ct: CancellationToken
-        ) =
+    let getSymbolUsesInProjects (symbol: FSharpSymbol, projects: Project list, onFound: Document -> range -> CancellableTask<unit>) =
         match projects with
-        | [] -> Task.CompletedTask
+        | [] -> CancellableTask.singleton ()
```

Comment:
For future: maybe worth it to have a "CancellableTask.CompletedTask", to not create a singleton every time.